frappe/website/doctype/web_form/web_form.py

```diff
@@ -37,7 +36,9 @@ def use_meta_fields(self):
 				continue
 
 			for prop in docfield_properties:
-				if df.fieldtype==meta_df.fieldtype and prop != "idx":
+				if df.fieldtype==meta_df.fieldtype and prop not in ("idx",
+					"reqd", "default", "description", "default", "options",
+					"hidden", "read_only", "label"):
```

Comment:
Why did you ignore these properties? "Label" was one of the customizations that users wanted in Web Forms.